### ISSUE TITLE
[BUG] Get-PnPSiteTemplate: The property or field 'ServerRelativePath' has not been initialized

### ISSUE THREAD

**Issue by KellyDJones** [pnp/powershell#110]
### Reporting an Issue or Missing Feature
Error when getting site template with credentials (not certificate)

### Actual behavior 
Get-PnPSiteTemplate: The property or field 'ServerRelativePath' has not been initialized. It has not been requested or the request has not been executed. It may need to be explicitly requested.

### Steps to reproduce the behavior
```
$cred = Get-Credential
Connect-PnpOnline -Url {siteUrl} -Credentials $cred
Get-PnPSiteTemplate -Out {filename}.xml -PersistBrandingFiles -PersistPublishingFiles -IncludeAllPages
```

### What is the version of the Cmdlet module you are running?
```
ModuleType Version    PreRelease Name                                PSEdition
---------- -------    ---------- ----                                --------- 
Manifest   0.3.33     nightly    PnP.PowerShell                      Core,Desk
```

### Which operating system/environment are you running PnP PowerShell on?
- [ X ] Windows

**Comment by erwinvanhunen** [comment]
We fixed the issue. It will be available 0.3.34-nightly.

**Comment by KellyDJones** [comment]
Thank you!
